### PR TITLE
use edraak images from GCR

### PR DESCRIPTION
**Pull request by devalih**

### CODE REVIEW

File: repo.sh [hunk 1]
```diff
@@ -29,10 +29,14 @@ repos=(
...
     "git@github.com:Edraak/marketing-site.git"
     "git@github.com:Edraak/edraak-programs.git"
+    "git@github.com:Edraak/edraak-2019-theme.git"
+    "git@github.com:Edraak/shared-devstack-configs.git"
 )
 
 repo_alternative_directory=(
 	"https://github.com/Edraak/edraak-platform.git,edx-platform"
+	"git@github.com:Edraak/edraak-2019-theme.git,edx-themes/edraak-2019-theme"
+    "git@github.com:Edraak/shared-devstack-configs.git,src/edxapp-envs"
```
Comment (OmarEdraak) [inline repo.sh]:
```suggestion
      "git@github.com:Edraak/shared-devstack-configs.git,src/edxapp-envs"
```

File: provision-edraak.sql [hunk 2]
```diff
@@ -1,2 +1,3 @@
 CREATE DATABASE IF NOT EXISTS edraakprograms;
 CREATE DATABASE IF NOT EXISTS marketingsite;
+-- TODO add some edraak fixtures
```
Comment (OmarEdraak) [inline provision-edraak.sql]:
```suggestion
-- TODO add some edraak fixtures

```

File: edraak.mk [hunk 3]
```diff
@@ -9,10 +9,10 @@ edraak.build.all:
 	$(MAKE) edraak.build.marketing
 
 edraak.build.programs:
-	docker build -t edraak-devstack-programs -f ../edraak-programs/Dockerfile ../edraak-programs
+	docker build -t eu.gcr.io/openedx-231314/edraak/progs -f ../edraak-programs/Dockerfile ../edraak-programs
```
Comment (OmarEdraak) [inline edraak.mk]: Great. Let's undo the edraak hacks on `make pull` and make it look like this:

```
pull: ## Update Docker images
	docker-compose pull --parallel
```

Comment (devalih) [reply edraak.mk]: done

**Pull request merged**